pkg/controller/codes/bulk_issue_test.go

```diff
@@ -38,7 +38,7 @@ func TestShowBulkIssueCodes(t *testing.T) {
 	db.SaveRealm(realm, database.SystemTest)
 
 	config := &config.ServerConfig{}
-	h, err := render.NewTest(ctx, "", t)
+	h, err := render.NewTest(ctx, "../../../cmd/server/assets", t)
```

Comment:
Use project.Root instead please

Reply:
aha that's exactly what I was looking for